instrumentation/github.com/gin-gonic/gin/otelgin/option.go

```diff
@@ -70,6 +77,13 @@ func WithFilter(f ...Filter) Option {
 	})
 }
 
+// WithGinFilter adds a filter to the list of filters used by the handler.
```

Comment:
```suggestion
// WithGinFilter adds a filter to the list of gin filters used by the handler.
// If any filter indicates to exclude a request then the request will not be
// traced. All gin and net/http filters must allow a request to be traced for a Span to be created.
// If no filters are provided then all requests are traced.
// Filters will be invoked for each processed request, it is advised to make them
// simple and fast.
```

Could you also add the `All gin and net/http filters` mention in the `WithFilter` comment above?

Reply:
sure, let me do that right away.